scripts/gff_metaparser/conf/valid_structures.conf

```diff
@@ -150,6 +149,9 @@ gene/pseudogene/intron	IGNORE
 gene/trna/intron	IGNORE
 gene/sequence_variant	IGNORE
 
+## IGNORED STRANGE OR INVALID MODELS / PARTS
+pseudogene/three_prime_UTR	IGNORE:ignore_pseudogene_utr:UNSEEN
```

Comment:
could you make them just [always] "IGNORE", under no flags ?

Reply:
or do you prefer to have a flag to report them by default?

Reply:
If we set them to IGNORE outright, they are not going to be reported in the gff_stats.err log file right ? I think let's leave it for now. Its not something seen very often and reporting it then setting the flag in conf isn't difficult to do.

Reply:
makes sense. thanks!